irods/use-local-irods.sh

```diff
@@ -34,7 +34,7 @@ if [[ "${1}" == "--persist" ]]; then
         mkdir -p /home/${USER}/icat2/vault
         mkdir -p /home/${USER}/icat2/pgdata
     fi
-    # Create ${IRODS_HOST} container from irods v.4.1.8
+    # Create ${IRODS_HOST} container from irods v.4.2.6
```

Comment:
Did you intend to sneak in an irods upgrade with this PR?

Reply:
just wanted to get things to build
but you're right I guess that's poor form to sneak it in here
once we merge this https://github.com/hydroshare/hydroshare/pull/5053, the iRods upgrade will make it into Develop